buildlib/az-distro-release.yml

```diff
@@ -39,16 +39,18 @@ jobs:
       - bash: |
           set -eE
           ./autogen.sh
-          ./contrib/configure-release --with-cuda
+          mkdir pkg-build
```

Comment:
is it removed if already exists?

Reply:
Azure does clean checkout, so it won't exist here.